plugins/inputs/bond/bond_test.go

```diff
@@ -62,18 +34,68 @@ Link Failure Count: 0
 Permanent HW addr:
 `
 
+var sampleTestLACP = `
+Ethernet Channel Bonding Driver: v3.7.1 (April 27, 2011)
+
+Bonding Mode: IEEE 802.3ad Dynamic link aggregation
+Transmit Hash Policy: layer2 (0)
+MII Status: up
+MII Polling Interval (ms): 100
+Up Delay (ms): 0
+Down Delay (ms): 0
+
+802.3ad info
+LACP rate: fast
+Min links: 0
+Aggregator selection policy (ad_select): stable
+
+Slave Interface: eth0
+MII Status: up
+Speed: 10000 Mbps
+Duplex: full
+Link Failure Count: 2
+Permanent HW addr: 3c:ec:ef:5e:71:58
+Slave queue ID: 0
+Aggregator ID: 2
+Actor Churn State: none
+Partner Churn State: none
+Actor Churned Count: 2
+Partner Churned Count: 0
+
+Slave Interface: eth1
+MII Status: up
+Speed: 10000 Mbps
+Duplex: full
+Link Failure Count: 1
+Permanent HW addr: 3c:ec:ef:5e:71:59
+Slave queue ID: 0
+Aggregator ID: 2
+Actor Churn State: none
+Partner Churn State: none
+Actor Churned Count: 0
+Partner Churned Count: 0
+`
+
+var sampleSysMode = "802.3ad 5"
+var sampleSysSlaves = "eth0 eth1 "
+var sampleSysAdPorts = " 2 "
```

Comment:
Can we make those `const` please?
```suggestion
const sampleSysMode = "802.3ad 5"
const sampleSysSlaves = "eth0 eth1 "
const sampleSysAdPorts = " 2 "
```